### DISCUSSION TITLE
issue connecting turtles

### DISCUSSION THREAD

**Discussion by Johnnybon-2** [runi95/turtle-control-panel#18] [General]
![image](https://github.com/user-attachments/assets/20ed03ef-3aac-463d-b843-0d288941304e)
i dont know how to get the mentioned uuid to connect my turtle to the server im actively hosting

**Comment by runi95** [comment]
@Johnnybon-2 the `uuid` is just a randomly generated value received from the server that is sent back to the server when the turtle replies to messages to ensure the messages are received back again in the correct order and in response to the correct events. The line you're looking at simply sets the uuid to an empty string if no uuid is sent in as an argument to the function and can be safely ignored.

The only thing that needs to be changed to get the turtle up and running is the `connectionURL` which needs to be changed to your server's IP or left as localhost if you're just using this on a local server / singleplayer world.
